packages/neutron-sdk/src/stargate/query_dex.rs

```diff
@@ -0,0 +1,224 @@
+    QueryGetLimitOrderTrancheUserRequest, QueryGetPoolMetadataRequest, QueryGetPoolReservesRequest,
+    QueryParamsRequest, QueryPoolByIdRequest, QueryPoolRequest,
+};
+use crate::stargate::types_dex::{
+    AllInactiveLimitOrderTrancheRequest, AllInactiveLimitOrderTrancheResponse,
+    AllLimitOrderTrancheRequest, AllLimitOrderTrancheResponse, AllPoolMetadataRequest,
+    AllPoolMetadataResponse, AllPoolReservesRequest, AllPoolReservesResponse,
+    AllTickLiquidityRequest, AllTickLiquidityResponse, AllUserDepositsRequest,
+    AllUserDepositsResponse, AllUserLimitOrdersRequest, AllUserLimitOrdersResponse,
+    EstimateMultiHopSwapRequest, EstimateMultiHopSwapResponse, EstimatePlaceLimitOrderRequest,
+    EstimatePlaceLimitOrderResponse, GetInactiveLimitOrderTrancheRequest,
+    GetInactiveLimitOrderTrancheResponse, GetLimitOrderTrancheRequest,
+    GetLimitOrderTrancheResponse, GetPoolMetadataRequest, GetPoolMetadataResponse,
+    GetPoolReservesRequest, GetPoolReservesResponse, LimitOrderTrancheUserAllRequest,
+    LimitOrderTrancheUserAllResponse, LimitOrderTrancheUserRequest, LimitOrderTrancheUserResponse,
+    ParamsRequest, ParamsResponse, PoolByIdRequest, PoolRequest, PoolResponse,
+};
+use cosmwasm_std::{Deps, StdResult};
+
+const PARAMS_QUERY_PATH: &str = "/neutron.dex.Query/Params";
+const LIMIT_ORDER_TRANCHE_USER_QUERY_PATH: &str = "/neutron.dex.Query/LimitOrderTrancheUser";
+const LIMIT_ORDER_TRANCHE_USER_ALL_QUERY_PATH: &str = "/neutron.dex.Query/LimitOrderTrancheUserAll";
+const LIMIT_ORDER_TRANCHE_USER_ALL_BY_ADDRESS_QUERY_PATH: &str =
+    "/neutron.dex.Query/LimitOrderTrancheUserAllByAddress";
+const LIMIT_ORDER_TRANCHE_QUERY_PATH: &str = "/neutron.dex.Query/LimitOrderTranche";
+const LIMIT_ORDER_TRANCHE_ALL_QUERY_PATH: &str = "/neutron.dex.Query/LimitOrderTrancheAll";
+const USER_DEPOSITS_ALL_QUERY_PATH: &str = "/neutron.dex.Query/UserDepositsAll";
+const TICK_LIQUIDITY_ALL_QUERY_PATH: &str = "/neutron.dex.Query/TickLiquidityAll";
+const INACTIVE_LIMIT_ORDER_TRANCHE_QUERY_PATH: &str =
+    "/neutron.dex.Query/InactiveLimitOrderTranche";
+const INACTIVE_LIMIT_ORDER_TRANCHE_ALL_QUERY_PATH: &str =
+    "/neutron.dex.Query/InactiveLimitOrderTrancheAll";
+const POOL_RESERVES_ALL_QUERY_PATH: &str = "/neutron.dex.Query/PoolReservesAll";
+const POOL_RESERVES_QUERY_PATH: &str = "/neutron.dex.Query/PoolReserves";
+const ESTIMATE_MULTI_HOP_SWAP_QUERY_PATH: &str = "/neutron.dex.Query/EstimateMultiHopSwap";
+const ESTIMATE_PLACE_LIMIT_ORDER_QUERY_PATH: &str = "/neutron.dex.Query/EstimatePlaceLimitOrder";
+const POOL_QUERY_PATH: &str = "/neutron.dex.Query/Pool";
+const POOL_BY_ID_QUERY_PATH: &str = "/neutron.dex.Query/PoolByID";
+const POOL_METADATA_QUERY_PATH: &str = "/neutron.dex.Query/PoolMetadata";
+const POOL_METADATA_ALL_QUERY_PATH: &str = "/neutron.dex.Query/PoolMetadataAll";
+
+pub fn get_params(deps: Deps, req: ParamsRequest) -> StdResult<ParamsResponse> {
```

Comment:
What do you think of adding doc comments for each method which describes what does it do?